Apps/Sandcastle/.jshintrc

```diff
@@ -1,12 +1,69 @@
 {
-  "extends": "../.jshintrc",
-  "unused": false,
-  "esversion": 10,
+  "esversion": 11,
   "predef": [
     "JSON",
     "require",
     "console",
     "Sandcastle",
     "Cesium"
-  ]
-}
+  ],
+  "bitwise": false,
+  "camelcase": false,
+  "curly": true,
+  "eqeqeq": true,
+  "forin": true,
+  "freeze": true,
+  "immed": true,
+  "latedef": false,
+  "newcap": true,
+  "noarg": true,
+  "noempty": true,
+  "nonbsp": true,
+  "nonew": true,
+  "plusplus": false,
+  "quotmark": false,
+  "undef": true,
+  "unused": false,
+  "strict": true,
+  "asi": false,
+  "boss": false,
+  "debug": false,
+  "eqnull": false,
+  "moz": false,
+  "evil": false,
+  "expr": false,
+  "funcscope": false,
+  "globalstrict": false,
+  "iterator": false,
+  "lastsemic": false,
+  "laxbreak": false,
+  "laxcomma": false,
+  "loopfunc": false,
+  "multistr": true,
+  "noyield": false,
+  "notypeof": false,
+  "proto": false,
+  "scripturl": false,
+  "shadow": false,
+  "sub": false,
+  "supernew": false,
+  "validthis": false,
+  "browser": true,
+  "browserify": false,
+  "couch": false,
+  "devel": true,
+  "dojo": false,
+  "jasmine": false,
+  "jquery": false,
+  "mocha": true,
```

Comment:
I know this was `true` in the previous configuration, but should the [`mocha` option](https://jshint.com/docs/options/#mocha) be `false`? @mramato Do you know why this option may have been enabled before?

Reply:
Most of these options probably aren't needed at all and can just be removed entirely. I vaguely remember that there was a reason for us to set them all explicitly at some point, but that was so early in development that there isn't even GitHub history for it :open_mouth: 

Some of these probably don't even exist anymore/  All the options are on https://jshint.com/docs/options/, though it looks like it doesn't include default values.

One approach would be delete everything in this file except esversion and then just put stuff back if we noticed it putting up warnings we don't want.  Since this is _just_ used for Sandcastle, it may also be better to play a little looser than what we do in the actual Cesium source.